src/SignalRServiceExtension/Config/OptionsSetup.cs

```diff
@@ -38,6 +40,7 @@ public OptionsSetup(IConfiguration configuration, ILoggerFactory loggerFactory,
         public void Configure(ServiceManagerOptions options)
         {
             options.ConnectionString = configuration[connectionStringKey];
+            options.ServiceEndpoints = configuration.GetEndpoints(connectionStringKey).ToArray();
```

Comment:
add a log when cx is using multi endpoint hub context, and tell cx config ServiceTransportType is ignored

Reply:
If user use `Transient` mode, then multiple-endpoint is disable.